tmt/steps/report/__init__.py

```diff
@@ -9,14 +9,20 @@
 from tmt.options import option
 from tmt.plugins import PluginRegistry
 from tmt.steps import Action
+from tmt.utils import field
 
 if TYPE_CHECKING:
     import tmt.cli
 
 
 @dataclasses.dataclass
 class ReportStepData(tmt.steps.StepData):
-    pass
+    output_log: bool = field(
+        default=True,
+        option=('--output-log / --no-output-log'),
```

Comment:
Maybe `include-onput-log`? `output-log` alone sounds like there would be an output log created by the plugin rather than including output in XML.